sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -436,12 +521,13 @@ abstract class AbstractSync(
         isRemovingItem: suspend (TimelineItem) -> Boolean,
         finalBlock: List<TimelineItem>,
         relevantTimeline: List<TimelineItem>,
-        restoresDefaultState: Boolean
+        restoresDefaultState: Boolean,
+        virtualIDs: Map<TimelineItem, String> = mapOf()
```

Comment:
docs for new param

Reply:
fafafc8c6b5332803aa70add51f61e003561ff7d